plugins/core_user.py

```diff
@@ -64,7 +64,7 @@ def desktop(inp, nick=None, db=None, notice=None):
 @hook.command(autohelp=False)
 def greeting(inp, nick=None, db=None, notice=None):
     "greet <message | @ person> -- Shows a users Greeting."
-    if nick == 'kimi' or nick == 'Kali_' or nick == 'kelp':
```

Comment:
Should be:
`if nick in {'kimi', 'Kali_', 'kelp'}:`
@inexist3nce thoughts?

Reply:
no!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Reply:
Proof of the tangible [performance benefit.](https://kelp.ml/u/axut.png).

Reply:
@FrozenPigs you seem to be forgetting the fact that such a change would be detrimental to performance in a case where there would be only one nick that is being excluded from this command.

Reply:
we could improve the `{'kimi', 'Kali_', 'kelp'}` by moving it to the config and adding a command to ignore specific "commands + nicks", like `.cignore kelp intro`.

would modifying `@hook.command` be too overkill? taigabot could have granular control over commands.

Reply:
I don't know if going that far is worth the effort really, it's not often I need to block people from a single command, and in this case the end user wanted to be blocked in the first place.